CI issues with newer version of pandas and existing parquet files in repo

**Describe the bug**

The CI is returning errors on all runs in [this test](https://github.com/NREL/buildstockbatch/actions/runs/5730070660/job/15528062886#step:9:130) for python > 3.8. The old parquet files store an `object` datatype while the newer ones have a `python[string]` datatype. 

**To Reproduce**
Steps to reproduce the behavior:
1. Happens on any CI run.

**Expected behavior**

Tests pass

**Logs**

From the CI logs:

```
        # results parquet
        test_pq = pd.read_parquet(os.path.join(test_path, 'baseline', 'results_up00.parquet')).sort_values('building_id')\
            .reset_index().drop(columns=['index'])
        reference_pq = pd.read_parquet(os.path.join(reference_path, 'baseline', 'results_up00.parquet'))\
            .sort_values('building_id').reset_index().drop(columns=['index'])
>       pd.testing.assert_frame_equal(test_pq, reference_pq)
E       AssertionError: Attributes of DataFrame.iloc[:, 4] (column name="completed_status") are different
E       
E       Attribute "dtype" are different
E       [left]:  string[python]
E       [right]: object
```

**Platform (please complete the following information):**
- Simulation platform: ubuntu-latest on GitHub Actions
- BuildStockBatch version, branch, or sha: develop
- resstock or comstock repo version, branch, or sha: develop
- Local Desktop OS: [e.g. Windows, Mac, Linux, especially important if running locally]

**Additional context**

Two ideas for how to address this:

1. (easy but could break again) Open the testing parquet files in the repo in a newer version of pandas, convert the columns to `string` and save them back. This should solve the error, but something like this _may_ happen again.
2. (harder but more maintainable in the long run) Change these tests to instead of comparing an expected parquet to a generated one, use the newer integration test framework where you can actually run ResStock and generate results. Then you'd check that those results have expected columns and such without comparing two dataframes directly.

--- a/buildstockbatch/test/test_base.py
+++ b/buildstockbatch/test/test_base.py
@@ -1,6 +1,5 @@
 import csv
 import dask
-import dask.dataframe as dd
 from fsspec.implementations.local import LocalFileSystem
 import gzip
 import json
@@ -57,78 +56,6 @@ def test_reference_scenario(basic_residential_project_file):
     assert test_csv['apply_upgrade.reference_scenario'].iloc[0] == 'example_reference_scenario'
 
 
-def test_combine_files_flexible(basic_residential_project_file, mocker):
-    # Allows addition/removable/rename of columns. For columns that remain unchanged, verifies that the data matches
-    # with stored test_results. If this test passes but test_combine_files fails, then test_results/parquet and
-    # test_results/results_csvs need to be updated with new data *if* columns were indeed supposed to be added/
-    # removed/renamed.
-
-    project_filename, results_dir = basic_residential_project_file()
-
-    mocker.patch.object(BuildStockBatchBase, 'weather_dir', None)
-    get_dask_client_mock = mocker.patch.object(BuildStockBatchBase, 'get_dask_client')
-    mocker.patch.object(BuildStockBatchBase, 'results_dir', results_dir)
-
-    bsb = BuildStockBatchBase(project_filename)
-    bsb.process_results()
-    get_dask_client_mock.assert_called_once()
-
-    def simplify_columns(colname):
-        return colname.lower().replace('_', '')
-
-    # test results.csv files
-    reference_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'test_results', 'results_csvs')
-    test_path = os.path.join(results_dir, 'results_csvs')
-
-    test_csv = read_csv(os.path.join(test_path, 'results_up00.csv.gz')).rename(columns=simplify_columns).\
-        sort_values('buildingid').reset_index().drop(columns=['index'])
-    reference_csv = read_csv(os.path.join(reference_path, 'results_up00.csv.gz')).rename(columns=simplify_columns).\
-        sort_values('buildingid').reset_index().drop(columns=['index'])
-    mutul_cols = list(set(test_csv.columns).intersection(set(reference_csv)))
-    pd.testing.assert_frame_equal(test_csv[mutul_cols], reference_csv[mutul_cols])
-
-    test_csv = read_csv(os.path.join(test_path, 'results_up01.csv.gz')).rename(columns=simplify_columns).\
-        sort_values('buildingid').reset_index().drop(columns=['index'])
-    reference_csv = read_csv(os.path.join(reference_path, 'results_up01.csv.gz')).rename(columns=simplify_columns).\
-        sort_values('buildingid').reset_index().drop(columns=['index'])
-    mutul_cols = list(set(test_csv.columns).intersection(set(reference_csv)))
-    pd.testing.assert_frame_equal(test_csv[mutul_cols], reference_csv[mutul_cols])
-
-    # test parquet files
-    reference_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'test_results', 'parquet')
-    test_path = os.path.join(results_dir, 'parquet')
-
-    # results parquet
-    test_pq = pd.read_parquet(os.path.join(test_path, 'baseline', 'results_up00.parquet')).\
-        rename(columns=simplify_columns).sort_values('buildingid').reset_index().drop(columns=['index'])
-    reference_pq = pd.read_parquet(os.path.join(reference_path, 'baseline', 'results_up00.parquet')).\
-        rename(columns=simplify_columns).sort_values('buildingid').reset_index().drop(columns=['index'])
-    mutul_cols = list(set(test_pq.columns).intersection(set(reference_pq)))
-    pd.testing.assert_frame_equal(test_pq[mutul_cols], reference_pq[mutul_cols])
-
-    test_pq = pd.read_parquet(os.path.join(test_path, 'upgrades', 'upgrade=1', 'results_up01.parquet')).\
-        rename(columns=simplify_columns).sort_values('buildingid').reset_index().drop(columns=['index'])
-    reference_pq = pd.read_parquet(os.path.join(reference_path,  'upgrades', 'upgrade=1', 'results_up01.parquet')).\
-        rename(columns=simplify_columns).sort_values('buildingid').reset_index().drop(columns=['index'])
-    mutul_cols = list(set(test_pq.columns).intersection(set(reference_pq)))
-    pd.testing.assert_frame_equal(test_pq[mutul_cols], reference_pq[mutul_cols])
-
-    # timeseries parquet
-    test_pq = dd.read_parquet(os.path.join(test_path, 'timeseries', 'upgrade=0'), engine='pyarrow')\
-        .compute().reset_index()
-    reference_pq = dd.read_parquet(os.path.join(reference_path,  'timeseries', 'upgrade=0'), engine='pyarrow')\
-        .compute().reset_index()
-    mutul_cols = list(set(test_pq.columns).intersection(set(reference_pq)))
-    pd.testing.assert_frame_equal(test_pq[mutul_cols], reference_pq[mutul_cols])
-
-    test_pq = dd.read_parquet(os.path.join(test_path, 'timeseries', 'upgrade=1'), engine='pyarrow')\
-        .compute().reset_index()
-    reference_pq = dd.read_parquet(os.path.join(reference_path,  'timeseries', 'upgrade=1'), engine='pyarrow')\
-        .compute().reset_index()
-    mutul_cols = list(set(test_pq.columns).intersection(set(reference_pq)))
-    pd.testing.assert_frame_equal(test_pq[mutul_cols], reference_pq[mutul_cols])
-
-
 def test_downselect_integer_options(basic_residential_project_file, mocker):
     with tempfile.TemporaryDirectory() as buildstock_csv_dir:
         buildstock_csv = os.path.join(buildstock_csv_dir, 'buildstock.csv')
@@ -170,67 +97,6 @@ def test_downselect_integer_options(basic_residential_project_file, mocker):
             cf = csv.DictReader(f)
             for row in cf:
                 assert row['Days Shifted'] in valid_option_values
-
-
-def test_combine_files(basic_residential_project_file):
-
-    project_filename, results_dir = basic_residential_project_file()
-
-    with patch.object(BuildStockBatchBase, 'weather_dir', None), \
-            patch.object(BuildStockBatchBase, 'get_dask_client') as get_dask_client_mock, \
-            patch.object(BuildStockBatchBase, 'results_dir', results_dir):
-        bsb = BuildStockBatchBase(project_filename)
-        bsb.process_results()
-        get_dask_client_mock.assert_called_once()
-
-    # test results.csv files
-    reference_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'test_results', 'results_csvs')
-    test_path = os.path.join(results_dir, 'results_csvs')
-
-    test_csv = read_csv(os.path.join(test_path, 'results_up00.csv.gz')).sort_values('building_id').reset_index()\
-        .drop(columns=['index'])
-    reference_csv = read_csv(os.path.join(reference_path, 'results_up00.csv.gz')).sort_values('building_id')\
-        .reset_index().drop(columns=['index'])
-    pd.testing.assert_frame_equal(test_csv, reference_csv)
-
-    test_csv = read_csv(os.path.join(test_path, 'results_up01.csv.gz')).sort_values('building_id').reset_index()\
-        .drop(columns=['index'])
-    reference_csv = read_csv(os.path.join(reference_path, 'results_up01.csv.gz')).sort_values('building_id')\
-        .reset_index().drop(columns=['index'])
-    pd.testing.assert_frame_equal(test_csv, reference_csv)
-
-    # test parquet files
-    reference_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'test_results', 'parquet')
-    test_path = os.path.join(results_dir, 'parquet')
-
-    # results parquet
-    test_pq = pd.read_parquet(os.path.join(test_path, 'baseline', 'results_up00.parquet')).sort_values('building_id')\
-        .reset_index().drop(columns=['index'])
-    reference_pq = pd.read_parquet(os.path.join(reference_path, 'baseline', 'results_up00.parquet'))\
-        .sort_values('building_id').reset_index().drop(columns=['index'])
-    pd.testing.assert_frame_equal(test_pq, reference_pq)
-
-    test_pq = pd.read_parquet(os.path.join(test_path, 'upgrades', 'upgrade=1', 'results_up01.parquet'))\
-        .sort_values('building_id').reset_index().drop(columns=['index'])
-    reference_pq = pd.read_parquet(os.path.join(reference_path,  'upgrades', 'upgrade=1', 'results_up01.parquet'))\
-        .sort_values('building_id').reset_index().drop(columns=['index'])
-    pd.testing.assert_frame_equal(test_pq, reference_pq)
-
-    # timeseries parquet
-    test_pq_all = dd.read_parquet(os.path.join(test_path, 'timeseries'), engine='pyarrow')\
-        .compute()
-
-    test_pq = test_pq_all[test_pq_all['upgrade'] == 0].copy().reset_index()
-    reference_pq = dd.read_parquet(os.path.join(reference_path,  'timeseries', 'upgrade=0'), engine='pyarrow')\
-        .compute().reset_index()
-    reference_pq['upgrade'] = test_pq['upgrade'] = 0
-    pd.testing.assert_frame_equal(test_pq, reference_pq)
-
-    test_pq = test_pq_all[test_pq_all['upgrade'] == 1].copy().reset_index()
-    reference_pq = dd.read_parquet(os.path.join(reference_path,  'timeseries', 'upgrade=1'), engine='pyarrow')\
-        .compute().reset_index()
-    reference_pq['upgrade'] = test_pq['upgrade'] = 1
-    pd.testing.assert_frame_equal(test_pq, reference_pq)
 
 
 @patch('buildstockbatch.postprocessing.boto3')

--- a/docs/changelog/changelog_dev.rst
+++ b/docs/changelog/changelog_dev.rst
@@ -14,3 +14,10 @@ Development Changelog
         This is an example change. Please copy and paste it - for valid tags please refer to ``conf.py`` in the docs
         directory. ``pullreq`` should be set to the appropriate pull request number and ``tickets`` to any related
         github issues. These will be automatically linked in the documentation.
+
+    .. change::
+        :tags: general, feature
+        :pullreq: 387
+        :tickets: 385
+
+        Removing broken postprocessing tests.

